examples/sql_bank/main.go

```diff
@@ -49,35 +82,9 @@ func moveMoney(db *sql.DB) {
 		if err != nil {
 			log.Fatal(err)
 		}
-		if _, err = tx.Exec("SET TRANSACTION ISOLATION LEVEL SERIALIZABLE"); err != nil {
-			log.Fatal(err)
-		}
-		rows, err := tx.Query("SELECT id, balance FROM accounts WHERE id IN ($1, $2)", from, to)
```

Comment:
We should really keep the single query to retrieve both balances. It's slow now, but will be more efficient in a few days than separate queries. It's good to know that doing the separate queries fixes the performance.